addon/templates/components/frost-numeric-select.hbs

```diff
@@ -0,0 +1,77 @@
+  {{hookable-input
+    ariaActiveDescendant=ariaActiveDescendant
+    ariaAutocomplete=ariaAutocomplete
+    ariaExpanded=ariaExpanded
+    ariaMultiselectable=ariaMultiselectable
+    ariaOwns=ariaOwns
+    autocapitalize=autocapitalize
+    autocorrect=autocorrect
+    autofocus=autofocus
+    hook=hook
+    class=class
+    disabled=disabled
+    form=form
+    hook=hook
+    isHookEmbedded=true
+    maxlength=maxlength
+    placeholder=placeholder
+    required=required
+    selectionDirection=selectionDirection
+    tabindex=tabindex
+    title=title
+    type=type
+    value=value
+
+    change=_eventProxy.change
+    click=_eventProxy.click
+    contextMenu=_eventProxy.contextMenu
+    doubleClick=_eventProxy.doubleClick
+    drag=_eventProxy.drag
+    dragEnd=_eventProxy.dragEnd
+    dragEnter=_eventProxy.dragEnter
+    dragLeave=_eventProxy.dragLeave
+    dragOver=_eventProxy.dragOver
+    dragStart=_eventProxy.dragStart
+    drop=_eventProxy.drop
+    focusIn=_eventProxy.focusIn
+    focusOut=_eventProxy.focusOut
+    input=(action '_onInput')
+    keyDown=_eventProxy.keyDown
+    keyPress=(action '_onKeyPress')
+    key-up='keyUp'
+    mouseDown=_eventProxy.mouseDown
+    mouseEnter=_eventProxy.mouseEnter
+    mouseLeave=_eventProxy.mouseLeave
+    mouseMove=_eventProxy.mouseMove
+    mouseUp=_eventProxy.mouseUp
+    submit=_eventProxy.submit
+    touchCancel=_eventProxy.touchCancel
+    touchEnd=_eventProxy.touchEnd
+    touchMove=_eventProxy.touchMove
+    touchStart=_eventProxy.touchStart
+
+    onEnter=_eventProxy.enter
+    onEscape=_eventProxy.escape-press}}
+  <div class='container'>
+    {{#if isArrowVisible}}
+      {{frost-button
+        hook='arrowDown'
+        class='plusButton'
+        icon='numeric-chevron'
+        pack='frost'
+        onClick= (action '_onPlusClick')
+      }}
+      {{frost-button
+        hook='arrowDown'
+        class='minusButton'
+        icon='numeric-chevron'
+        pack='frost'
+        onClick= (action '_onMinusClick')
+      }}
+    {{/if}}
+  </div>
+  {{#if showError}}
```

Comment:
We _may_ need a generic error - e.g. "Hey, this field needs a value greater than 20" and the numeric error - I'd say the numeric error (`Input must be numeric`) would take precedence, so maybe we can do a computed property `_errorMessage` where the `errorMessage` (external) and the `_numericErrorMessage` (internal) are inputs to the computed property (CP) and we return the `_numericErrorMessage` if it's present.

I think that could remove the need for `showError` and the condition could just be `{{#if _errorMessage}}`